### DISCUSSION TITLE
The compact header that shows up when scrolling down a page is missing the repository info

### DISCUSSION THREAD

**Discussion by StingyJack** [community/community#45745] [Accessibility]
### Select Topic Area

Product Feedback

### Body

When I click on an inbox notification and am taken to the issue comment, the page is often scrolled down far enough that the compact header is shown. In the example below, I have no idea what org or repository I am looking at unless I scroll all the way back to the top.  There is plenty of empty space available on the compact header, please put the org/repo on it somewhere. 
![image](https://user-images.githubusercontent.com/15381181/215360939-f846b9f4-f1a0-437f-ac9c-4c8f43f672e8.png)

**Comment by cehfisher** [comment]
Hi @StingyJack - this issue appears to be resolved with recent theme changes that include the repo name in the sticky header. Could you please confirm this is true for you? If it does resolve the issue, please close out this issue. Thank you! (cc @queenofcorgis)

**Reply by StingyJack** [reply]
@cehfisher  - I dont think I see it in here. It looks the same to me. 
![image](https://github.com/community/community/assets/15381181/3c356611-1927-42d9-a5e4-4277e28575a7)
Am I just missing it?

**Reply by cehfisher** [reply]
My apologies, it looks like the sticky header currently only shows the issue title, not the repo where it was posted. We will file this with our Accessibility team so that it can be reviewed and will follow-up here with any additional questions or updates. Thanks again!

**Comment by cehfisher** [comment]


**Reply by StingyJack** [reply]
Please don't feel like you have to spend the time to post a long comment that really says nothing. 

To be honest I would have rather not wasted my time reading it.
